extra/provision.sh

```diff
@@ -1,9 +1,34 @@
 #!/bin/bash
 #
-# Facebook CTF: Provision script for vagrant dev environment
+# fbctf provisioning script
 #
-# Usage: ./provision.sh [dev | prod] [path_to_code] [path_to_destination]
+# Usage: usage [-h|--help] [PARAMETER [ARGUMENT]] [PARAMETER [ARGUMENT]] ...
 #
+# Parameters:
+#   -h, --help            Shows this help message and exit.
+#   -m MODE, --mode MODE  Mode of operation. Default value is dev
+#   -c TYPE, --cert TYPE  Type of certificate to use. Default value is self
+#
+# Arguments:
+#   dev    Provision will run in development mode. Certificate will be self-signed.
```

Comment:
You should say that these three arguments correspond to MODE and the next two correspond to TYPE.